src/components/Table/TableFormatters/AccountInfoFormatter.vue

```diff
@@ -72,15 +68,13 @@ export default {
 .detail-content {
   max-height: 150px;
   overflow-y: auto;
-  min-width: 350px;
+  min-width: 300px;
 }
 
 .detail-item {
   border-bottom: 1px solid #EBEEF5;
   padding: 5px 0;
   margin-bottom: 0;
-  display: flex;
-  justify-content: space-between;
 
   &:hover {
     background-color: #F5F7FA;
```

Comment:
There don't seem to be any major issues with this code snippet. However, some things you might consider refining include:

- Adding a return statement at `export default {`, which makes it clearer that the component is complete.
   
- For consistency, ensure consistent spacing within tags like `<span>` and `.el-tag`
  
Remember, these are just suggestions based on the current knowledge limit I have as of now (Oct 2021 - Jul 2025), so if there's something more specific about what needs improvement please let me know!